plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/MysqlDaoProvider.java

```diff
@@ -28,11 +28,14 @@
     private final ResourceGroupsDao dao;
 
     @Inject
-    public MysqlDaoProvider(DbResourceGroupConfig config)
+    public MysqlDaoProvider(FlywayMigration migration, DbResourceGroupConfig config)
     {
+        requireNonNull(migration, "migration is null");
         requireNonNull(config, "config is null");
         MysqlDataSource dataSource = new MysqlDataSource();
         dataSource.setURL(requireNonNull(config.getConfigDbUrl(), "resource-groups.config-db-url is null"));
```

Comment:
We always require this property to be non-null, so we should annotate it win `@NotNull` and get rid of the `requireNonNull()` calls.  Up to you, but if you decide to make that change, please do it in a separate commit.

Reply:
I am going to make a follow up PR to add migrations for Oracle once this goes in. I will make this change in the follow up PR.